Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/internal/ConfigDataContainer.kt

```diff
@@ -13,12 +13,15 @@ data class ConfigDataContainer(
     override val localOffset: Duration,
     override val configType: ConfigData.Type
 ) : ConfigData, ConfigMapping by mappedConfig {
+
+    override val isDeviceTimeCorrect: Boolean
+        get() = !isDeviceTimeCheckEnabled || localOffset.abs() < ConfigData.DEVICE_TIME_GRACE_RANGE
```

Comment:
disabled but corect time: true | true -> true 
disabled but wrong time: true | true -> true 
enabled but correct time: false | true -> true 
enabled but wrong time: false | false -> false

Looks good - am just wondering how this plays out if the testers disable the clock-check the tester menu. What's the priority then?

Reply:
If the test menu option "disable time check" is enabled, then `isDeviceTimeCheckEnabled` will return `false`.